internal/controller/controllers/spoke_client_cache_test.go

```diff
@@ -0,0 +1,114 @@
+package controllers
+
+import (
+	"errors"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/spoke_k8s_client"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	testKubeconfigSecretName      = "test-secret"
```

Comment:
Any reason these can't be scoped in the `Context`?

Specifically this same constant is defined in the `"HypershiftAgentServiceConfig reconcile"` describe block.
If they were different values this might be confusing.